pkg/virtualnode/resource_manager_test.go

```diff
@@ -0,0 +1 @@
+package virtualnode
```

Comment:
No tests? 🤨

Reply:
yeah, plan to add after initial review, adding :D